source/basic.tex

```diff
@@ -1183,13 +1183,15 @@
 or before its use in a namespace enclosing its namespace.
 
 \pnum
-A name used in the definition of a function following the function's
+In the definition of a function that is a member of namespace \tcode{N}
+(where, only for the purpose of
```

Comment:
This change looks like a significant improvement to me already, but can we also remove this parenthetical? The global scope *is* a namespace ([basic.scope.namespace]/3), so we don't need to say that it's one here just for exposition.